snaps/how-to/provide-signature-insights.md

```diff
@@ -0,0 +1,139 @@
+---
+description: Provide insights to your users in MetaMask's signature confirmation window.
```

Comment:
```suggestion
description: Provide insights to your users in MetaMask's signature confirmation flow.
```

This seems slightly more accurate given the insight is in a modal and not in the actual confirmation window.